doc/interactors.md

```diff
@@ -76,3 +79,46 @@ Accordion("Categories").exists()
 - `index`: _number_ = the 0 based index of this accordion with in a
   pane set. So for example `Accordion("Users").has({ index: 2 })`
   would assert that the "Users" accordion was 3rd in its accordion set.
+
+---
+
+#### Pane
+
+The pane element
+
+##### Synopsis
+
+``` javascript
+import { Pane } from '@folio/stripes-testing';
+
+Pane("User Pane").is({ visible: true });
+```
+
+##### Locator
+
+Accordions are identified by their title. For example:
+
+```javascript
+Pane("User Pane").exists();
```

Comment:
Panes do not usually have the word "Pane" in them. I'd use `Pane("Users")`

Reply:
I was thinking that too, but wondered if `"User"` was too vague.

Reply:
I think in the context of someone using stripes, it's not. Our examples should reflect a plausible usage

Reply:
I see your point though. Maybe `Pane("Search Results')` would be less ambiguous?

Reply:
Oh, that is good. Switching to that.